File: solutions/python/armstrong-numbers/1/armstrong_numbers.py
def is_armstrong_number(number):
    number_string = str(number)
    digits = [int(digit) for digit in number_string]
    digits_sq = []
    for digit in digits:
        digits_sq.append(digit ** len(number_string))
    
    return sum(digits_sq) == number
    
    

File: solutions/java/armstrong-numbers/1/src/main/java/ArmstrongNumbers.java
class ArmstrongNumbers {

    boolean isArmstrongNumber(int numberToCheck) {
     String number = String.valueOf(numberToCheck);
        int powerSum = 0;
        int exponent = number.length();
    
        for (char digitCharacter : number.toCharArray()){
            int digit = Character.getNumericValue(digitCharacter);
            powerSum += (int) Math.pow(digit, exponent);  
        }
        return powerSum == numberToCheck;      

    }

}
